changelog/3298.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix parfive using the same loop once it's closed once.
```

Comment:
Can this changelog entry be more descriptive to capture how users might actually encounter this bug?